de.tudarmstadt.ukp.wikipedia.api/pom.xml

```diff
@@ -22,7 +22,7 @@
 	<parent>
 		<groupId>de.tudarmstadt.ukp.wikipedia</groupId>
 		<artifactId>de.tudarmstadt.ukp.wikipedia</artifactId>
-		<version>1.2.0-SNAPSHOT</version>
+		<version>1.2.1-SNAPSHOT</version>
```

Comment:
Why would this PR change the version of the master branch?

Reply:
@reckart I use that to test a new dependency in a repo called json wikipedia, which we use for processing wikipedia dumps. I will revert to the original version